pkg/operatormanager/operatormanager.go

```diff
@@ -480,6 +487,7 @@ func (m *OperatorManager) ensurePodEnvironmentConfigMap(ctx context.Context, nam
 	}
 	if err := m.Get(ctx, ns, &v1.ConfigMap{}); err == nil {
 		// configmap already exists, nothing to do here
+		// we will update the configmap with the correct S3 config in the postgres controller
```

Comment:
We should shift the logic of updating *pod environment* `ConfigMap` to `OperatorManager` since it's part of `OperatorManager`. In another branch.

Reply:
#160